src/main/resources/schema.sql

```diff
@@ -0,0 +1,21 @@
+drop table if exists cameras cascade;
```

Comment:
Do you really need to drop table every time at the application start?
Maybe it's better to do `create table if not exist cameras...` and drop it manually if needed.